code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -13,6 +13,8 @@
 	var/generate_render_target = TRUE
 	///integer: blend mode to apply to the render relay in case you dont want to use the plane_masters blend_mode
 	var/blend_mode_override
+	///reference: current relay this plane is utilizing to render
+	var/atom/movable/render_plane_relay/relay
```

Comment:
where does this gets cleared?

Reply:
likely when the client disconnects I'd assume. Nothing else holds a reference to it and it gets autocleared by the game's GC.

Reply:
Assuming this gets deleted when the client gets deleted.

Reply:
Wouldn't be better to null it before having to GC it? Not making any assumptions here I'm just curious

Reply:
If you look at circuits, this is what they did before and it was unnecessary and ended up clogging the files a lot. You don't have to qdel or destroy something that'll get gc'd on its own